tests/ops/fixtures/saas/shopify_fixtures.py

```diff
@@ -100,3 +104,153 @@ def shopify_dataset_config(
+    )
+    customer = customers_response.json()
+    assert customers_response.ok
+    error_message = f"customer with email {shopify_erasure_identity_email} could not be added to Shopify"
+    poll_for_existence(
+        customer_exists,
+        (shopify_erasure_identity_email, shopify_secrets),
+        error_message=error_message,
+    )
+
+    # Create Order
+    body = {
+        "order": {
+            "email": shopify_erasure_identity_email,
+            "fulfillment_status": "fulfilled",
+            "send_receipt": True,
+            "financial_status": "paid",
+            "send_fulfillment_receipt": True,
+            "line_items": [
+                {
+                    "product_id": 6923717967965,
+                    "name": "Short leeve t-shirt",
+                    "title": "Short sleeve t-shirt",
+                    "price": 10,
+                    "quantity": 1,
+                }
+            ],
+        }
+    }
+    orders_response = requests.post(
+        url=f"{base_url}/admin/api/2022-07/orders.json", json=body, headers=headers
+    )
+
+    assert orders_response.ok
+    order = orders_response.json()
+    order_id = order["order"]["id"]
+
+    # Get Blog
+    blogs_response = requests.get(
+        url=f"{base_url}/admin/api/2022-07/blogs.json", headers=headers
+    )
+    assert blogs_response.ok
+    blog = blogs_response.json()["blogs"][1]
+
+    blog_id = blog["id"]
+    # Create Article
+    body = {
+        "article": {
+            "title": "Test Article",
+            "author": firstName,
+        }
+    }
+    articles_response = requests.post(
+        url=f"{base_url}/admin/api/2022-07/blogs/{blog_id}/articles.json",
+        json=body,
+        headers=headers,
+    )
+    assert articles_response.ok
+    article = articles_response.json()
+    article_id = article["article"]["id"]
+
+    # Create Comment
+    body = {
+        "comment": {
+            "body": "I like comments\nAnd I like posting them *RESTfully*.",
+            "author": firstName,
+            "email": shopify_erasure_identity_email,
+            "ip": faker.ipv4_private(),
+            "blog_id": blog_id,
+            "article_id": article_id,
+        }
+    }
+    comments_response = requests.post(
+        url=f"{base_url}/admin/api/2022-07/comments.json", json=body, headers=headers
+    )
+    assert comments_response.ok
+    comment = comments_response.json()
+
+    yield customer, order, blog, article, comment
+
+    # Deleting order and article after verifying update request
+    order_delete_response = requests.delete(
+        url=f"{base_url}/admin/api/2022-07/orders/{order_id}.json",
+        headers=headers,
+    )
+    assert order_delete_response.ok
+    article_delete_response = requests.delete(
+        url=f"{base_url}/admin/api/2022-07/articles/{article_id}.json",
+        headers=headers,
+    )
+    assert article_delete_response.ok
```

Comment:
maybe i'm mising something, but is there a reason we don't delete/clean up the `customer` that we've created for the test here?

Reply:
Yes basically we don't have a delete customer endpoint in Shopify API, that is why we can't delete newly created customer.

Reply:
hmm, OK - so do we end up just creating a new customer every time we run a test? could that get unwieldy after a while? just wondering if shopify provides any insight into whether having too many customers could become a problem. i guess it's something they should be able to handle if they don't offer any API to delete a customer.

anyway, not a blocker, especially since i don't think there's a whole lot we can do about this. but i am a _little_ bit concerned we may start running into problems after months of running tests and creating customers each time.

Reply:
I had discussion with Adrian related to it too and we were rooting for approach of not creating customer in every call first too and get already created customer only but we had a problem that data was already masked just after our first call and there was practically no use of our assertions for it so we decided to go for creating customer instead. Since we created a new Shopify store with no data and there is no limit of creating customers as far as we researched so we opted for this solution.

Reply:
got it - thanks for walking through that @HamzaWaseemOnBench !